docs/project/about.md

```diff
@@ -59,15 +59,39 @@ list from there with "The Pyodide development team" like in the example below:
 - Stack Overflow: [stackoverflow.com/questions/tagged/pyodide](https://stackoverflow.com/questions/tagged/pyodide)
 - Discord: [Pyodide Discord](https://dsc.gg/pyodide)
 
-## Donations
+## Institutional and Financial Support
 
-We accept donations to the Pyodide project at
-[opencollective.com/pyodide](https://opencollective.com/pyodide). All donations
-are processed by the [Open Source Collective](https://www.oscollective.org/) -- a
-nonprofit organization that acts as our fiscal host.
+Pyodide is a community-driven project that has benefited from the support of various institutions, grants, and employers who have allocated resources and time to its development.
 
-Funds will be mostly spent to organize in-person code sprints and to cover
-infrastructure costs for distributing packages built with Pyodide.
+### Current Support
+
+| Institution                                                                                                          | Contributor(s)      | Contribution Type       | Period       |
+| -------------------------------------------------------------------------------------------------------------------- | ------------------- | ----------------------- | ------------ |
+| Quansight Labs                                                                                                       | Agriya Khetarpal    | Employer-sponsored time | 2022–Present |
+| Chan Zuckerberg Initiative (CZI) [2022-316713](https://blog.scientific-python.org/scientific-python/2022-czi-grant/) | Agriya Khetarpal    | Research grant          | 2022–Present |
```

Comment:
The CZI grant was a two-year grant awarded at the end of 2022, so I'd guess this should read 2023-2024? @agriyakhetarpal?

Reply:
Yes, the grant was awarded in 2022, however, the remuneration started to be dispersed from 2024 onwards when I joined and has ended this month. I've been contributing as a volunteer for some time now :)

I think the 

```suggestion
| Chan Zuckerberg Initiative (CZI) [2022-316713](https://blog.scientific-python.org/scientific-python/2022-czi-grant/) | Agriya Khetarpal    | Research software grant          | 2024–2025 |
```

Reply:
Yes, the grant was awarded in 2022, however, the remuneration started to be dispersed from 2024 onwards when I joined and has ended this month. I've been contributing as a volunteer for some time now :)

I think this should help:

```suggestion
| Chan Zuckerberg Initiative (CZI) [2022-316713](https://blog.scientific-python.org/scientific-python/2022-czi-grant/) | Agriya Khetarpal    | Research software grant          | 2024–2025 |
```